core/integration/module_up_test.go

```diff
@@ -99,8 +103,6 @@ func (ModuleSuite) TestDaggerUp(ctx context.Context, t *testctx.T) {
 		},
 	}
 
-	t = t.WithTimeout(3 * time.Minute)
```

Comment:
I think this was here because this test has a slight tendency to misbehave (at higher rates than others). When it fails, it's nicer to have it fail after 3 minutes, instead of getting killed in github actions.

Reply:
yep, happy to bring this back, i have 0 recollection of removing it tbh, this may be a raw typo.

Reply:
oh wait, i remember what's going on here: this is extraneous, there's another t.WithTimeout call on line 25.